src/components/app/routes/loaders/rootLoader.js

```diff
@@ -230,22 +237,29 @@ export const makeEnterpriseLearnerOffersQuery = (enterpriseUuid) => ({
  * @returns
  */
 const fetchBrowseAndRequestConfiguration = async (enterpriseUuid, userEmail) => {
-  const results = await Promise.all([
-    fetchSubsidyRequestConfiguration(enterpriseUuid),
-    fetchCouponCodeRequests({
-      enterpriseUUID: enterpriseUuid,
-      userEmail,
-    }),
-    fetchLicenseRequests({
-      enterpriseUUID: enterpriseUuid,
-      userEmail,
-    }),
-  ]);
-  return {
-    subsidyRequestConfiguration: results[0].data,
-    couponCodeRequests: results[1].data,
-    licenseRequests: results[2].data,
-  };
+  try {
+    const results = await Promise.all([
+      fetchSubsidyRequestConfiguration(enterpriseUuid),
+      fetchCouponCodeRequests({
```

Comment:
[suggestion] Should we use the `try/catch` within `fetchSubsidyRequestConfiguration` specifically if that's the only API request throwing the 404? AFAIK (may be incorrect), I don't believe `coupon-code-requests` or `license-requests` throw a 404, even if there's no subsidy request configuration. Probably don't need to have special error handling around requests to not require the special treatment?